app/controllers/attendance_record_amendments_controller.rb

```
@@ -0,0 +1,23 @@
+class AttendanceRecordAmendmentsController < ApplicationController
+  authorize_resource
+
+  def new
+    @attendance_record_amendment = AttendanceRecordAmendment.new
+  end
+
+  def create
+    @attendance_record_amendment = AttendanceRecordAmendment.new(attendance_record_amendment_params)
+    if @attendance_record_amendment.save
+      student = Student.find(params[:attendance_record_amendment][:student_id])
+      redirect_to new_attendance_record_amendment_path, notice: "#{student.name}'s attendance record has been amended."
+    else
+      render :new, alert: 'Something went wrong. Please try again.'
```

please surface the error to the user.